### PR TITLE
Render GeoJson in features.

### PR DESCRIPTION
**Pull request by DaoyuT**
- [x] Render arbitrary GeoJSON stored in Feature in Firebase
- [x] Use default style of layer for styling

Screenshot:
Screen Shot 2020-08-10 at 1 13 26 AM

### CODE REVIEW

File: web-ng/src/app/shared/converters/firebase-data-converter.ts [hunk 1]
```diff
@@ -158,12 +162,23 @@ export class FirebaseDataConverter {
...
+        location,
+      };
+    } else if (feature instanceof GeoJsonFeature) {
+      const { layerId, geoJson } = feature;
+      return {
+        layerId,
+        geoJson,
+      };
+    } else {
+      const { layerId } = feature;
```
Comment (gino-m) [inline web-ng/src/app/shared/converters/firebase-data-converter.ts:179]: If the application itself is generating features of unknown types, we should probably throw an exception here. Wdyt?

**Comment by DaoyuT** [comment]
> Great work, really excited to see GeoJSON support working! Iiuc this impl only updates GeoJson features when the map is initialized, but not on remote changes. Will you implement that bit in a follow-up PR?

What do you mean? I tried adding/updating features in firebase, and then website reflects the update without me refreshing it. Is that what you meant?

File: web-ng/src/app/shared/converters/firebase-data-converter.ts (hunk 1 above)
Comment (DaoyuT) [reply web-ng/src/app/shared/converters/firebase-data-converter.ts]: Done.

**Comment by gino-m** [comment]
> > Great work, really excited to see GeoJSON support working! Iiuc this impl only updates GeoJson features when the map is initialized, but not on remote changes. Will you implement that bit in a follow-up PR?
> 
> What do you mean? I tried adding/updating features in firebase, and then website reflects the update without me refreshing it. Is that what you meant?

Sorry, you're right; I see you're subscribing in ngOnViewInit, not actually adding/removing features directly there. Please disregard!

**Pull request merged**